docs/how-to-guides/develop-standard-flow.md

```diff
@@ -0,0 +1,312 @@
+# Develop standard flow
```

Comment:
I think we can have a develop-a-flow parent folder

Reply:
added